Skills/Skill/Pregnancy.gd

```diff
@@ -0,0 +1,17 @@
+extends SkillBase
+
+func _init():
+	id = Skill.Pregnancy
+
+func getVisibleName():
+	return "Pregnancy"
```

Comment:
How about "Reproduction"

Reply:
Procreation or reproduction :thinking:

Reply:
waaait. How about Fertility skill. Fertility skill sounds perfect I think, even though it's a name of a buff already I think, who cares x3